### ISSUE TITLE
Cannot login to iOS Standard Notes App with Lockdown Mode enabled

### ISSUE THREAD

**Issue by mydewji** [standardnotes/forum#3051]
**Describe the bug**
I cannot login to iOS Standard Notes App with Lockdown Mode enabled. After inputting username/password the app freezes.

**To Reproduce**
1. Enable [lockdown](https://support.apple.com/en-ca/guide/iphone/iph049680987/ios#:~:text=on%20Lockdown%20Mode-,Go%20to%20Settings%20%3E%20Privacy%20%26%20Security%20%3E%20Lockdown%20Mode%2C%20then,tap%20Turn%20On%20Lockdown%20Mode.) mode on iPhone
2. Install/Launch standard notes (not logged into any account)
3. Attempt to login to existing account (also fails on account creation)
4. App freezes with no error presented to user

**Expected behavior**
User successfully signed in.

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: iPhone 14
 - OS: iOS 16.4.1
 - Browser n/a using Standard Notes App from App store
 - Version 3.158.1 (627)

**Additional context**
If I disable Lockdown mode I am able to login. I can then re-enable it and continue to used the signed in app (sync still works).

**Comment by effieeee** [comment]
We apologize that this issue took a very long while to look into, though it looks like on recent updates with iOS 16.6, the app now appears as an option to be excluded when Lockdown Mode is active. (**Settings > Privacy & Security > Lockdown Mode > Configure Web Browsing**)

May I ask if you can confirm that this is the case on your end as well?

**Comment by mydewji** [comment]
I can confirm there is an option to exclude it. I have left it turned on as the workaround I used previously still seems to work (disable lockdown, login, re-enable lockdown)

**Comment by at-xa** [comment]
Hello. Please consider re-opening this issue. The workaround of disabling lockdown for sn --> logging in to sn --> re-enabling lockdown and using sn, is no longer viable as of iOS17 and the latest current standard notes iOS version. 
(Unless someone can confirm otherwise.)

Lockdown is a nice piece of hardening/security work by apple so it would be cool to leave this issue open for any discussion for security minded users of standard notes to have regarding getting this to work again.

**Comment by effieeee** [comment]
Hi @yokusha, thanks for letting us know that there seems to be another change in behavior with this after iOS 17. However, since Apple also doesn't fully disclose what particular technologies are affected or inhibited by Lockdown Mode (understandably for security reasons), I'm afraid this also makes it much more difficult to diagnose the issue and come up with a definite solution.

In the meantime, I wanted to confirm if adding it as an exception under lockdown mode also doesn't help in your case? (please refer to [my comment above](https://github.com/standardnotes/forum/issues/3051#issuecomment-1658890314))

**Comment by at-xa** [comment]
Yes, the app works when giving it special exception from lockdown. 

Yet I hope the team still leaves this issue open so interested users can discuss ideas of what code can be contributed to make it work without the lockdown exception.


Thank you.

**Comment by EskimoOneFour** [comment]
The lockdown mode bypass (Settings: Privacy: Lockdown Mode: Configure Web Browsing: Standard Notes (toggled OFF) does NOT work since updating my iPhone to iOS 17.4.1. I have tried uninstalling and reinstalling the Standard Notes app, but nothing works. 

Details:
It fails at the 2FA auth code check every time by freezing grey with the grey button below the login details stuck on "Signing in..."

**Comment by effieeee** [comment]
Hi @EskimoOneFour, the team is aware of how the sign in workflow is inhibited by Lockdown Mode even when the app is added an exception in iOS 17.4. Unfortunately there isn't a fix for this yet, but it looks like it's possible to sign in with Lockdown Mode disabled temporarily, then you may turn it back on after signing in, and syncing should still proceed on the app 🙏

**Comment by locainin** [comment]
Just wanted to say im experiencing the same issue, a quick reddit search brought me here. Hopefully resolved in the future but i understand it can be challenging:)

**Comment by billgraham5** [comment]
I reproduced this bug on 17.5.  The exception process continues not to work.

**Comment by billgraham5** [comment]
@effieeee , is this bug expected to apply to the iOS app connecting to the self hosting server as well?

**Comment by yurybialykh** [comment]
Same issue iOS 17.5.1 & SN 3.194.11 (1103) - cannot login with Lockdown mode enabled, adding SN as exception doesn't work as well. App is stuck at "signing in" stage. I think there might be not a small overlap of groups of people who use Lockdown mode in iOS and SN for note taking.

**Comment by yurybialykh** [comment]
However, turning Lockdown mode off, signing into SN and turning Lockdown mode back on works as a workaround.

**Comment by edmundloo** [comment]
I'm also experiencing this issue. I don't have this problem with other apps; the workaround is frustrating. I would greatly appreciate a fix.

**Comment by matusf** [comment]
Hi, I'm on iOS 18.1 and I tried to turn off the lockdown mode, signing in, and turning it back on. I'm successfully signed in, but the notes are not syncing. I can only see the ones that loaded right after I signed in when lockdown mode was turned off. No new ones (created on other device) are displayed.

**Comment by alex27riva** [comment]
Hi everyone, I'm on iPhone 15 with iOS 18.6.2 and I still have this issue when lockdown mode is enabled.

![Image](https://github.com/user-attachments/assets/27c1224a-df49-44cb-8a9f-b8b0ae0d9e12)
